### PR TITLE
WMA from TV

### PR DESCRIPTION
**Pull request by seannowotny**
WMA which behaves identically to the WMA found in TV

### CODE REVIEW

File: technical/indicators/indicators.py [hunk 1]
```diff
@@ -1208,3 +1208,27 @@ def PMAX(dataframe, period=10, multiplier=3, length=12, MAtype=1, src=1):  # noq
     df.fillna(0, inplace=True)
 
     return df
+
+
+def tv_wma(series: Series, length: int) -> Series:
```
Comment (xmatthias) [inline technical/indicators/indicators.py:1213]: For consistency, this should take a dataframe - and maybe use a default parameter for length (ideally the default tradingview uses?)

As it's then a dataframe, no longer a series, a parameter `field='close'` or `src='close'` to (default) pick the right column wil be beneficial too.

Comment (seannowotny) [reply technical/indicators/indicators.py]: Good idea, see latest commit for changes

Comment (xmatthias) [reply technical/indicators/indicators.py]: Now i'm curious however - did you investigate how / why it's different to `ta.WMA(dataframe, 9)` (or `ta.WMA(dataframe, 10)`?

i can SEE that the numbers are different - but i don't really understand how / why

Comment (seannowotny) [reply technical/indicators/indicators.py]: I'm not sure, I noticed that the WMA in tradingview draws a different graph than the one in talib so I simply translated the pinescript code.

**Review by xmatthias — APPROVED** [review]
LGTM :+1:

**Pull request merged**